service/create/service.go

```diff
@@ -540,22 +540,40 @@ func (s *Service) addFunc(obj interface{}) {
 		return
 	}
 
-	// Create policy
 	bucketName := s.bucketName(cluster)
 
-	var policy resources.NamedResource
-	var policyErr error
+	// Create master IAM policy
+	var masterPolicy resources.NamedResource
+	var masterPolicyErr error
 	{
-		policy = &awsresources.Policy{
-			ClusterID: cluster.Spec.Cluster.Cluster.ID,
-			KMSKeyArn: kmsKey.Arn(),
-			S3Bucket:  bucketName,
-			AWSEntity: awsresources.AWSEntity{Clients: clients},
+		masterPolicy = &awsresources.Policy{
+			ClusterID:  cluster.Spec.Cluster.Cluster.ID,
```

Comment:
In some other operators we created key methods for data that is often accessed. That maybe helps at some point reducing the hustle to find out where what data is located within a structure. One day it also might help refactoring or changing the data's location within the structure.